src/main/java/com/openlattice/chronicle/constants/EdmConstants.java

```diff
@@ -40,11 +42,14 @@ private EdmConstants() {
     public static final FullQualifiedName MODEL_FQN       = new FullQualifiedName( "vehicle.model" );
     public static final FullQualifiedName DATE_TIME_FQN   = new FullQualifiedName( "ol.datetime" );
     public static final FullQualifiedName START_DATE_TIME = new FullQualifiedName( "ol.datetimestart" );
+    public static final FullQualifiedName END_DATE_TIME   = new FullQualifiedName( "ol.datetimeend" );
+    public static final FullQualifiedName RECORDED_DATE_TIME_FQN = new FullQualifiedName( "ol.recordeddate" );
```

Comment:
Could you run code reformat? (Code -> Reformat Code). You would need to configure formatting settings on IntelliJ though.  

**How set up on IntelliJ**

1. Save the file at https://github.com/openlattice/openlattice/blob/master/eclipse-code-format-settings.xml on your desktop
2. Open Preferences. Select code style -> Java -> Import Scheme -> Eclipse XML Profile